src/permissions.js

```diff
@@ -2,12 +2,23 @@ import memoize from 'lodash.memoize'
 
 const ANY_ADDRESS = '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF'
 
-const APP_MANAGER_ROLE = {
-  name: 'Manage apps',
-  id: 'APP_MANAGER',
-  params: [],
-  bytes: '0xb6d92708f3d4817afc106147d969e229ced5c46e65e0a5002a0d391287762bd0',
-}
+const KERNEL_ROLES = [
+  {
+    name: 'Manage apps',
+    id: 'APP_MANAGER',
```

Comment:
I'd change this to be `APP_MANAGER_ROLE` as all the other roles we're displaying have the `_ROLE` suffix.